apps/api/src/modules/resume/resume.service.ts

```diff
@@ -14,4 +18,31 @@ export class ResumeService {
     }
     return this.repository.createResume(postResumeDto, actor, user.userId)
```

Comment:
you didn't use actor in this method please remove. I didn't check this last time